polytope/polytope.py

```diff
@@ -290,6 +290,32 @@ def intersect(self, other, abs_tol=ABS_TOL):
 
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
+    def translation(self, d):
+        """Translate the Polytope by a given vector.
+
+        Consult L{polytope.polytope._translate} for implementation details.
+
+        @type d: 1d array
+        @param d: The translation vector.
+
+        @rtype: L{Polytope}
+        @return: A translated copy of the Polytope.
+        """
+        newpoly = self.copy()
+        _translate(newpoly, d)
+        return newpoly
+
+    def rotation(self, u, v, theta=None):
+        """Rotate the Polytope. Only simple rotations are implemented at this
```

Comment:
I might be missing something, but I think that [the comment above](https://github.com/tulip-control/polytope/pull/39/files/bf65d334ed373bc26fe14ac9bf16faca900af9a4) that `u` and `v` are not defined is redundant with my earlier comment that the documentation of hidden function `_rotate` should be available from that of the class method `rotation`.